recipes-core/images/xenclient-initramfs-image.bb

```diff
@@ -27,6 +28,15 @@ IMAGE_LINGUAS = "en-us"
 
 inherit openxt-image
 
+NO_RECOMMENDATIONS = "1"
+BAD_RECOMMENDATIONS += "busybox-syslog"
+
 PACKAGE_REMOVE = " \
     kernel-image-* \
 "
+post_rootfs_shell_commands() {
+    rm -f ${IMAGE_ROOTFS}/sbin/udhcpc;
+    rm -f ${IMAGE_ROOTFS}/sbin/ldconfig;
+    rm -rvf ${IMAGE_ROOTFS}/usr/lib/opkg;
```

Comment:
It looks like this comes from dom0's initramfs `POSTPROCESS`, I do not see `/usr/lib/opkg` present in the stubdomain initramfs. `opkg` seems to install `/var/lib/opkg` and `/var/cache/opkg` though.

Reply:
according to `du -cxsh`:
/usr/lib/opkg is 784K - lots of alternatives
/var/lib/opkg is 556K
/var/cache/opkg is 4.0K

If you remove /var/lib/opkg, you get a python backtrace when OE tries to look at /var/lib/opkg/status.  However something cleans up /var/lib/opkg and /var/cache/opkg afterwards.

/usr/lib/opkg requires manual cleanup

Reply:
Scratch my earlier comment, I mistakenly looked in the stubdomain initramfs instead, which has that same post-process already as mentioned...
`var/lib/opkg` and `/var/cache/opkg` would be populated by `IMAGE_FEATURE += "package-management"`, which the initramfs does not have.